### ISSUE TITLE
3.9 alpha5: Site is broken when update from rolled back version after updating theme

### ISSUE THREAD

**Issue by Mai-Saad** [wp-media/wp-rocket#3883]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version => Yes
- Used the search feature to ensure that the bug hasn’t been reported before =>Yes

**Describe the bug**
The latest theme is not displayed on site when updating to 3.9 from rolled back version after switching theme

**To Reproduce**
Precondition:
1- Install and activate 3.9
2- Activate RUCSS (while nothing is enabled in preload)
3- Visit the home page

1. In tools roll back to the previous version
2. Switch theme and revisit home page
4. Update to 3.9 (RUCSS is enabled) and revisit the home page
5. Broken theme is displayed

**Expected behavior**
The new theme is displayed

**Screenshots**

https://user-images.githubusercontent.com/76941962/117117530-ecd14b80-ad8f-11eb-8bf8-de478d5ebefb.mp4

**Additional context**
- Manual Clear Used CSS will fix the problem 
- Used settings: http://snippi.com/s/5vuk5ou

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by Mai-Saad** [comment]
The same will occur with the following scenario:
Clear used CSS to start from a clean state
1- RUCSS is enabled and the active theme is twenty twenty
2- visit the home page
3- disable RUCSS then change the theme
4- Enable RUCSS and revisit the home page => page is broken

**Comment by piotrbak** [comment]
The problem here is related to the tree shaker afaik. It's happening because the theme authors used wrong markup for the `content`:
```
#nav-fixed span::before,
#nav-fixed span:after {
	content: '"';
	position: relative;
}
```
It's rewritten by us to:
```
#nav-fixed span::before,
#nav-fixed span:after {
	content: """;
	position: relative;
}
```
Note the double-quotes. The wrong markup is used, but we should guard against this, as it's breaking the whole layout.

**Comment by Tabrisrp** [comment]
So the rollback and re-update is not a general issue?

**Comment by piotrbak** [comment]
@Tabrisrp Sorry, added it to the wrong issue, removing it and adding to the correct one.

**Comment by remyperona** [comment]
I tried the second scenario on the current new alpha, but I can't replicate it.

**Comment by Mai-Saad** [comment]
@Tabrisrp Yes, I can see too that scenario 2 is fixed on alpha 7 (using the same settings which caused this issue on alpha5) but Scenario 1 is still there.

**Comment by mostafa-hisham** [comment]
### Reproduce the issue :heavy_multiplication_x: 
i can see the issue

### Identify the root cause ✅ 
when we downgrade  from a version that has RUCSS to a version that does not have RUCSS 
then go back to the version with RUCSS 
Users may change the theme in between or do other actions that can affect the Used CSS files 
but the Used CSS is not updated because the feature is not in the old version


### Scope a solution ✅ 
we need to clear Used CSS files when we update to 3.9 to avoid this problem
we can delete the outdated Used CSS files in this condition 

https://github.com/wp-media/wp-rocket/blob/7c47b1e304692e6bb9c879fd32a8b2d8525c1455/inc/admin/upgrader.php#L386-L393
```
$cache_path           = rocket_get_constant( 'WP_ROCKET_CACHE_ROOT_PATH' );
			rocket_rrmdir( $cache_path . 'used-css' );
```
### Estimate the effort ✅

Effort `[XS]`

**Comment by piotrbak** [comment]
Closed with 3.11
